zerver/tests/test_users.py

```diff
@@ -1861,6 +1861,7 @@ def test_stream_recipient_info(self) -> None:
             default_bot_user_ids=set(),
             service_bot_tuples=[],
             all_bot_user_ids=set(),
+            topic_participant_user_ids=set(),
         )
```

Comment:
We should add a test for this fix. Basically, to check that the `wildcard_mentioned` flag is set even if notification settings are off for the user.

Reply:
#26506